doc/options.jax

```diff
@@ -8079,15 +8079,15 @@ Note 1番目の形式では、行全体がオプション指定に使われる
 						*'termwinkey'* *'twk'*
 'termwinkey' 'twk'	文字列	(既定では "")
 			ウィンドウについてローカル
-	ターミナルウィンドウ内で CTRL-W で始まるキーはコマンドである。他のキー
-	はウィンドウ内で動作しているジョブに送られる。
+	ターミナルウィンドウ内で CTRL-W コマンドを開始するキー。他のキーはウィ
+	ンドウ内で動作しているジョブに送られる。
 	<> 表記法を使うことができる。例えば: >
 		:set termwinkey=<C-L>
 <	この文字列は 1 つのキーストロークでなければならないが、マルチバイトで
 	も良い。
-	何も設定されていない CTRL-W が使われると、CTRL-W : はコマンドライン入
-	力になる。もしも 'termwinkey' が CTRL-L に設定されているなら、CTRL-L :
-	がコマンドライン入力になる。
+	未設定の場合は CTRL-W が使用されるため、CTRL-W : はコマンドラインに移
+	動する。'termwinkey' が CTRL-L に設定されている場合、CTRL-L : はコマン
```

Comment:
指摘というほどじゃないのですが、 CTRL-W : so that で、 CTRL-L : then なので、
「CTRL-Lに設定されている場合なら CTRL-L  : で コマンドラインに移動する」みたいな文章的な読みやすい変化があってもいいのかな、とかは思いました。(適切かは自信がない...)

Reply:
前半後半とも、`CTRL-～ : は` を `CTRL-～ : で` に変更しました。